code/game/gamemodes/vampire/vampire.dm

```diff
@@ -363,35 +381,46 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 		if(!(n in old_powers))
 			switch(n)
```

Comment:
Ideally these could use the current spell framework, and use an altered charge system to use blood.